### DISCUSSION TITLE
Where to download the dynamic version of the firebase frameworks

### DISCUSSION THREAD

**Discussion by Zeus64** [firebase/firebase-ios-sdk#10532] [Q&A] [answered]
I do not use xcode but Delphi development tool, and I would like to know where I can download the dynamic version of the firebase frameworks. as far as I understand the version under https://github.com/firebase/firebase-ios-sdk/releases are only the static framework right ?

**Comment by paulb777** [comment, marked as answer]
Firebase does not do a binary distribution of dynamically linked frameworks.

**Comment by jeffremer** [comment]
I'm curious if this is ever going to change. We rely on Firebase and Google libraries in a modularized project and want to stop using Cocoapods. The lack of dynamic frameworks is very limiting.

For instance, we have one module (the main app), that depends on `FirebaseAnalytics` and a handful of other Firebase libraries. We have another module that depends on `GoogleMobileAds`. Both of those have transitive dependencies to `GoogleAppMeasurement` and we get a ton of warnings in the console around duplicate symbols from `GoogleAppMeasurement`.

**Comment by paulb777** [comment]
We're planning to switch our binary distribution to dynamic frameworks in the next major release, targeted for sometime in the first half of 2024. See #8945

**Reply by jeffremer** [reply]
Hey, that's great news, thanks for the quick response @paulb777 👍
